test/e2e/apps/job.go

```diff
@@ -47,6 +47,14 @@ var _ = SIGDescribe("Job", func() {
 		ginkgo.By("Ensuring job reaches completions")
 		err = jobutil.WaitForJobComplete(f.ClientSet, f.Namespace.Name, job.Name, completions)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "failed to ensure job completion in namespace: %s", f.Namespace.Name)
+
+		ginkgo.By("Ensuring pods for job exist")
+		pods, err := jobutil.GetJobPods(f.ClientSet, f.Namespace.Name, job.Name)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "failed to get pod list for job in namespace: %s", f.Namespace.Name)
```

Comment:
Here can be just

`framework.ExpectNoError(err, "failed to get pod list for job in namespace: %s", f.Namespace.Name)`

instead for the readability.
It is fine to change only here without other places in this test module because of the PR scope.

Reply:
Thanks the comment. I have modified the code.